code/modules/sanity/breakdowns.dm

```diff
@@ -348,79 +348,253 @@
 	..()
 
 
+/datum/breakdown/common/power_hungry//work fine
+	name = "Power Hungry"
+	duration = 10 MINUTES
+	insight_reward = 20
+	restore_sanity_post = 80
+
+	start_messages = list("You think this doesn’t feel real... But reality hurts! Ensure that you will feel again!")
```

Comment:
This tells nothing user about what they should do. Also there is no repeated messages to clarify it.